### PR TITLE
[Bulk] Validation - Various fixes

### PR DESCRIPTION
**Pull request by vatmicro**
This PR fixes 28 validation issues.

### CODE REVIEW

File: iis/extensions/troubleshooting-application-request-routing/troubleshooting-502-errors-in-arr.md [hunk 1]
```diff
@@ -115,13 +115,13 @@ In this example, ARR started to receive data from the client but something went
 
 The HTTP 502.4 error with an associated error code of 0x00000000 generally indicates that all the members of the farm are either offline, or otherwise unreachable.
 
-[![Click to Expand](troubleshooting-502-errors-in-arr/_static/image8.png)](troubleshooting-502-errors-in-arr/_static/image7.png)
+[![A screenshot that shows a message of, no appropriate server could be found to route the request.](troubleshooting-502-errors-in-arr/_static/image8.png)](troubleshooting-502-errors-in-arr/_static/image7.png)
```
Comment (JustPies) [inline iis/extensions/troubleshooting-application-request-routing/troubleshooting-502-errors-in-arr.md:118]: The comma after "of" can be removed.

**Comment by vatmicro** [comment]
**Global effort to fix validation errors**

 @aczechowski   

The Content & Learning team is fixing build validation errors on docs.microsoft.com. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. I’d very much appreciate your timely review and feedback on these changes I’ve suggested. Thanks!

**Comment by aczechowski** [comment]
@vatmicro sorry, I'm not the author you're looking for
KenobiNotTheDroidsGIF

Looks like @rick-anderson is the author here for review? FYI I'm unsubscribing from the thread, assuming nothing further is needed from me on this PR. 😄

**Comment by vatmicro** [comment]
Global effort to fix validation errors

@Rick-Anderson 

The Content & Learning team is fixing build validation errors on docs.microsoft.com. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. I’d very much appreciate your timely review and feedback on these changes I’ve suggested. Thanks!

**Comment by jmart1428** [comment]
@Rick-Anderson Please let us know if you'd like to see any additional changes in this PR. If no changes are required, you are welcome to merge. Thank you for your feedback!

File: iis/get-started/whats-new-in-iis-10-version-1809/new-features-introduced-in-iis-10-1809.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 title: New Features Introduced  in IIS 10.0, version 1809
 author: shirhatti
 description: Describes the new features introduced in IIS 10.0 version 1809 and outlines the new binding flags and new compression API.
-ms.author: soshir
+ms.author: rick-anderson
```
Comment (wadepickett) [inline iis/get-started/whats-new-in-iis-10-version-1809/new-features-introduced-in-iis-10-1809.md:5]:
```suggestion
ms.author: johnhart

```

File: iis/get-started/whats-new-in-iis-10-version-1809/new-features-introduced-in-iis-10-1809.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 title: New Features Introduced  in IIS 10.0, version 1809
 author: shirhatti
```
Comment (wadepickett) [inline iis/get-started/whats-new-in-iis-10-version-1809/new-features-introduced-in-iis-10-1809.md:3]:
```suggestion
author: John-Hart

```

File: iis/extensions/web-playlists/iplaylistrequestinfo-getservervariable-method.md [hunk 4]
```diff
@@ -1,5 +1,6 @@
 ---
 title: IPlaylistRequestInfo::GetServerVariable Method
+Description: Learn how the GetServerVariable method returns a single server variable from a collection of server variables associated with the request.
```
Comment (wadepickett) [inline iis/extensions/web-playlists/iplaylistrequestinfo-getservervariable-method.md:3]:
```suggestion
description: Learn how the GetServerVariable method returns a single server variable from a collection of server variables associated with the request.
```

**Review by wadepickett — APPROVED** [review]

**Comment by jmart1428** [comment]
@wadepickett thanks so much for your help on these!

**Pull request merged**